test/test_special_elixirs.jl

```diff
@@ -18,6 +18,24 @@ const EXAMPLES_DIR = pkgdir(Trixi, "examples")
 #! format: noindent
 
 @testset "Convergence test" begin
+    @timed_testset "TreeMesh1D: elixir_diffusion_ldg.jl convergence" begin
+        # define initial condition here to avoid "world age" issue
+        function initial_condition_pure_diffusion_convergence_test(x, t, equation)
+            nu = diffusivity()
+            c = 0
+            A = 1
+            omega = 1
+            scalar = c + A * sin(omega * sum(x)) * exp(-nu * omega^2 * t)
+            return SVector(scalar)
+        end
```

Comment:
```suggestion
```
Looking at the other `convergence_test`s in this file, maybe we don't need to define the initial conditions here again?

Reply:
Sounds good to me.

Reply:
Looks like the "world age" error is still there, so I don't think removing the definition of the ICs will fix things either. I'll try defining it outside the macros; I don't want to move it yet to an `equations` file because it requires `diffusivity()` to be defined externally (see https://github.com/trixi-framework/Trixi.jl/issues/1503). 

If the CI fails again, would you be OK with just removing the convergence test from CI? We could at least add a short script to reproduce the convergence rates in this PR.

Reply:
Maybe @ranocha has an idea how to fix the world age issues?